com.unity.multiplayer.mlapi/Runtime/Configuration/NetworkConfig.cs

```diff
@@ -61,28 +61,10 @@ public class NetworkConfig
         internal Dictionary<uint, NetworkPrefab> NetworkPrefabOverrideLinks = new Dictionary<uint, NetworkPrefab>();
 
         /// <summary>
-        /// Amount of times per second the receive queue is emptied and all messages inside are processed.
+        /// The tickrate of network ticks. This value controls how often MLAPI runs user code and sends out data.
         /// </summary>
-        [Tooltip("The amount of times per second the receive queue is emptied from pending incoming messages")]
-        public int ReceiveTickrate = 64;
-
-        /// <summary>
-        /// Duration in seconds between network ticks.
-        /// </summary>
-        [Tooltip("Duration in seconds between network ticks")]
-        public float NetworkTickIntervalSec = 0.050f;
-
-        /// <summary>
-        /// The max amount of messages to process per ReceiveTickrate. This is to prevent flooding.
-        /// </summary>
-        [Tooltip("The maximum amount of Receive events to poll per Receive tick. This is to prevent flooding and freezing on the server")]
-        public int MaxReceiveEventsPerTickRate = 500;
-
-        /// <summary>
-        /// The amount of times per second internal frame events will occur, e.g. send checking.
-        /// </summary>
-        [Tooltip("The amount of times per second the internal event loop will run. This includes for example NetworkVariable checking.")]
-        public int EventTickrate = 64;
+        [Tooltip("The tickrate. This value controls how often MLAPI runs user code and sends out data. The value is in 'ticks per seconds' which means a value of 50 will result in 50 ticks being executed per second or a fixed delta time of 0.02.")]
+        public int TickRate = 60;
```

Comment:
I think we need to pay a little bit more attention to this default value. I can't really form a strong opinion atm but I think it should be different from 50 (physics/fixedupdate) and 60 (render/update). Maybe playing safe and going down to 20/24/30 would be a sensible default since we're trying to achieve co-op and even some open-world shooters pick 20 as a baseline? (I'm thinking Battlefield etc.)

Reply:
Using 30 now as the default.